website/docs/docs/deploy/deploy-jobs.md

```diff
@@ -67,9 +67,9 @@ To set your job's schedule, use the **Run on schedule** option to choose specifi
 
 Under **Timing**, you can either use customizable hours for jobs that need to run frequently throughout the day or exact intervals for jobs that need to run at specific times:
 
-- **Exact intervals** &mdash; Use this option to set how often your job runs, in hours. Enter a number between 1 and 23 to represent the interval between job runs. For example, if you set it to **Every 2 hours**, the job will run every 2 hours from midnight UTC. This option is useful if you need to run jobs multiple times per day at regular intervals.
+- **Intervals** &mdash; Use this option to set how often your job runs, in hours. Enter a number between 1 and 23 to represent the interval between job runs. For example, if you set it to **Every 2 hours**, the job will run every 2 hours from midnight UTC. This option is useful if you need to run jobs multiple times per day at regular intervals.
```

Comment:
Perhaps we can change the "Enter a number between..." line to be closer to the control where you specify what the interval is (always divisible by 24 hours).

Reply:
I'm confused by this part in general.
`Use this option to set how often your job runs, in hours. Enter a number between 1 and 23 to represent the interval between job runs.`
Aren't you selecting an interval from the drop-down rather than entering any kind of number? What am I missing here?

Reply:
great flag!